packages/integrations/components/integrationsManager/index.js

```diff
@@ -0,0 +1,144 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+// User Config.
+import userConfig from '@irvingjs/irving.config';
+// Redux.
+import { connect } from 'react-redux';
+import { actionHydrateComponents } from '../../actions';
+// Integration components.
+import GoogleAnalytics from '../googleAnalytics';
+
+/**
+ * Integrations Manager.
+ *
+ * A component that consumes a `irving/integrations` configuration from the
+ * REST API, Irving config, or persisted Redux state slice and renders the
+ * corresponding integrations on the client.
+ *
+ * @param {Object}   props - All props.
+ * @param {Object}   props.integrations - Config object for active integrations.
+ * @param {boolean}  props.isHydrated - Has the component map already been hydrated?
+ * @param {function} props.hydrateComponents - Redux action for updating the `componentMap` in the state tree.
+ * @param {Object}   props.hydratedComponents - The existing component key/prop pairs.
+ */
+const IntegrationsManager = ({
+  integrations,
+  isHydrated,
+  hydrateComponents,
+  hydratedComponents,
+}) => {
+  // Declare a map of available integration components with a corresponding key.
+  const componentMap = [
+    { key: 'googleAnalytics', el: GoogleAnalytics },
+  ];
+  // This check is run on server-side renders to ensure that the component hydration
+  // is ran a single time. After hydration, `isHydrated` is set to `true` and will
+  // only be re-run if the component props change during the component's lifecycle.
+  if (false === isHydrated) {
+    // Retrieve the keys available for render in the component map.
+    const keyMap = Object.keys(integrations);
+
+    // Create an array for hydrated component keys and props.
+    const components = [];
+
+    componentMap.forEach((component, index) => {
+      // Get the component's key.
+      const { key } = component;
+
+      if (- 1 < keyMap.indexOf(key)) {
+        // Retrieve the key/value pairs set for the integration and convert them into props.
+        // If a user config override exists, use it instead.
+        let props;
+        if ('integrations' in userConfig && key in userConfig.integrations) {
```

Comment:
Do you thin there'd be any use in allowing other irving packages to configure integrationgs? If so, you might look into using [`getConfigValues`](https://github.com/alleyinteractive/irving/blob/master/packages/core/config/irving/getValueFromConfig.js#L27) to grab the relevant values.

Reply:
I think there would be merit there. During dev I tried using `getConfigValues` initially to no avail. IIRC it would return as undefined, so I think some further investigation would be needed there.

Reply:
Hmmm ok, we can investigate separately.